source/s2n/s2n_tls_channel_handler.c

```diff
@@ -168,23 +168,30 @@ static const char *s_determine_default_pki_ca_file(void) {
     return NULL;
 }
 
+/* If s2n is already initialized, then we don't call s2n_init() or s2n_cleanup() ourselves */
+static bool s_s2n_initialized_externally;
+
 void aws_tls_init_static_state(struct aws_allocator *alloc) {
     (void)alloc;
     AWS_LOGF_INFO(AWS_LS_IO_TLS, "static: Initializing TLS using s2n.");
 
-    setenv("S2N_ENABLE_CLIENT_MODE", "1", 1);
```

Comment:
We don't need this anymore?

Reply:
S2N doesn't use that variable for anything anymore https://github.com/aws/s2n-tls/issues/771